apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -122,29 +160,50 @@ const FinishStudentAccount: React.FunctionComponent<{
             </>
           )}
         </div>
-        <TextField
-          name="displayName"
-          label={locale.display_name_eg()}
-          value={name}
-          placeholder={locale.coder()}
-          onChange={onNameChange}
-        />
-        <SimpleDropdown
-          name="userAge"
-          labelText={locale.what_is_your_age()}
-          size="m"
-          items={ageOptions}
-          selectedValue={age}
-          onChange={onAgeChange}
-        />
-        <SimpleDropdown
-          name="userState"
-          labelText={locale.what_state_are_you_in()}
-          size="m"
-          items={usStateOptions}
-          selectedValue={state}
-          onChange={onStateChange}
-        />
+        <div>
+          <TextField
+            name="displayName"
+            label={locale.display_name_eg()}
+            value={name}
+            placeholder={locale.coder()}
+            onChange={onNameChange}
+          />
+          {showNameError && (
+            <BodyThreeText className={style.errorMessage}>
+              {locale.display_name_error_message()}
+            </BodyThreeText>
+          )}
+        </div>
+        <div>
+          <SimpleDropdown
+            name="userAge"
+            labelText={locale.what_is_your_age()}
+            size="m"
+            items={ageOptions}
+            selectedValue={age}
+            onChange={onAgeChange}
+          />
+          {showAgeError && (
+            <BodyThreeText className={style.errorMessage}>
+              {locale.age_error_message()}
+            </BodyThreeText>
+          )}
+        </div>
+        <div>
+          <SimpleDropdown
```

Comment:
I know this isn't the change you're making in this PR but should the state dropdown be shown to all users? Or just US-based users?

Reply:
@dmcavoy Not sure? Currently there's a "I don't live anywhere listed here" option but that also seems a bit misleading for people who aren't in the U.S. altogether?

Reply:
Happy to make a ticket for this!

Reply:
This is the student account page right? My understanding was that we only show the state dropdown to people we think are in the US. We may want to confirm with platform.